packages/mock-block-dock/dev/dev-app.tsx

```diff
@@ -33,9 +33,9 @@ const DevApp: FunctionComponent = () => {
           Increment Entity ID
         </button>
 
-        <label htmlFor="reaodnly-input" style={{ marginLeft: 20 }}>
+        <label htmlFor="external-readonly-input" style={{ marginLeft: 20 }}>
```

Comment:
Do you need this `htmlFor`? The label is wrapping the input, which means it should be identified as belonging to the input automatically. 

It also avoids the `id` potentially clashing with an element in the block.

The same comment applies to the other readonly input.

Reply:
Added the `htmlFor` to fix an eslint error, which the label wrapping the input should actually fix... Will remove

Reply:
Updated in [828bd21](https://github.com/blockprotocol/blockprotocol/pull/488/commits/828bd2197ad5f074ec53c2f2491efbdc3b2b42d0)